ftw/ruleset.py

```diff
@@ -121,7 +120,7 @@ def __init__(self, raw_request=None,
             if 'Content-Type' in list(headers.keys()):
                 if headers['Content-Type'] == \
                    'application/x-www-form-urlencoded' and stop_magic is False:
-                    if ensure_str(unquote(self.data)) == self.data:
+                    if unquote(self.data) == self.data:
```

Comment:
Why this is not using util.ensure_str but other places are?

Reply:
I think because I was using py 3.9, and the method works both in `str` and `bytes`. But for compatibility with other versions, let's use ensure_str.